### DISCUSSION TITLE
Default setting for Item_Toggle?

### DISCUSSION THREAD

**Discussion by nielmot** [forntoh/LcdMenu#121] [Q&A] [answered]
I have a Item_Toggle item in my menu.  When toggled it will set a int variable to a 1 or 0 to enable/disable a setting in my code.  This setting will be saved and reloaded every time the program is run (after a reset, power cycle, etc).

 I have it successfully toggling on and off this variable in my callback function, so the menu is working.

However, every time the program loads, the value of the toggle menu item is set to OFF (0).  Is there a way to pre set this value to ON when it starts up?  This would be needed when my saved variable is set to 1.  

Thanks!
ps, I am not a professional programmer... go easy on me...

**Comment by forntoh** [comment, marked as answer]
In the setup block, you can find the specific menu item and set its value.

This might help

https://github.com/forntoh/LcdMenu/discussions/56#discussioncomment-5791075
